orca/service.go

```diff
@@ -90,16 +108,9 @@ func Register(s grpc.ServiceRegistrar, opts ServiceOptions) (*Service, error) {
 		return nil, fmt.Errorf("concrete type of provided grpc.ServiceRegistrar is %T, only supported type is %T", s, &grpc.Server{})
```

Comment:
Can we start with the API taking a `*grpc.Server` and change it in the future to a `grpc.ServiceRegistrar` when we can support it?

Reply:
Done.